src/program/snabbvmx/query/query.lua

```diff
@@ -98,15 +98,22 @@ local function print_counters (pid, dir)
   print(("   </%s>"):format(dir))
 end
 
+local named_program_root = engine.named_program_root
+
+function get_name_by_pid (pid)
```

Comment:
This can also be done with https://github.com/Igalia/snabb/blob/lwaftr/src/core/app.lua#L165 - just pass "true" as the first argument nd it'll give you back a table with pid's as the name.

It would be a bit slower than this approach though. If that's an important issue maybe this should be put in app.lua with the other name stuff it seems generic and useful enough.